### ISSUE TITLE
IncorrectCopyright doesn't work after git mv'ing

### ISSUE THREAD

**Issue by juippis** [pkgcore/pkgcheck#267]
```
joonas@astraea ~/git/devgentoo $ cd dev-libs/efl/
joonas@astraea ~/git/devgentoo/dev-libs/efl $ ls
efl-1.25.1-r10.ebuild  efl-1.25.1-r2.ebuild  Manifest  metadata.xml
joonas@astraea ~/git/devgentoo/dev-libs/efl $ git mv efl-1.25.1-r2.ebuild efl-1.25.1-r3.ebuild
joonas@astraea ~/git/devgentoo/dev-libs/efl $ edit efl-1.25.1-r3.ebuild 
joonas@astraea ~/git/devgentoo/dev-libs/efl $ git add efl-1.25.1-r3.ebuild 
joonas@astraea ~/git/devgentoo/dev-libs/efl $ pkgcommit -m "."
++ mktemp
+ tmpf=/tmp/tmp.hhUm8JKSfr
++ pkg
+ echo 'dev-libs/efl: .'
+ [[ -n . ]]
+ [[ -s /tmp/tmp.hhUm8JKSfr ]]
+ git commit -sS -F /tmp/tmp.hhUm8JKSfr
[master 863762254ad] dev-libs/efl: .
 1 file changed, 1 insertion(+)
 rename dev-libs/efl/{efl-1.25.1-r2.ebuild => efl-1.25.1-r3.ebuild} (99%)
+ rm -f /tmp/tmp.hhUm8JKSfr
joonas@astraea ~/git/devgentoo/dev-libs/efl $ cd ..
joonas@astraea ~/git/devgentoo/dev-libs $ cd ..
joonas@astraea ~/git/devgentoo $ pkgcheck scan --commits
dev-libs/efl
  RequiredUseDefaults: version 1.25.1-r3: profile: 'default/linux/amd64/17.1/desktop/gnome' (23 total) failed REQUIRED_USE: !opengl
  RequiredUseDefaults: version 1.25.1-r3: profile: 'default/linux/amd64/17.1/desktop/gnome' (23 total) failed REQUIRED_USE: gles2-only
  RequiredUseDefaults: version 1.25.1-r10: profile: 'default/linux/amd64/17.1/desktop/gnome' (19 total) failed REQUIRED_USE: !opengl
  RequiredUseDefaults: version 1.25.1-r10: profile: 'default/linux/amd64/17.1/desktop/gnome' (19 total) failed REQUIRED_USE: gles2-only
  StableRequest: version 1.25.1-r10: slot(0) no change in 60 days for unstable keywords: [ ~amd64, ~x86 ]
joonas@astraea ~/git/devgentoo $ grep -i copyright dev-libs/efl/efl-1.25.1-r3.ebuild 
# Copyright 1999-2020 Gentoo Authors
```

**Comment by radhermit** [comment]
What version of pkgcheck are you running? I can't seem to reproduce this using the latest git support.

**Comment by juippis** [comment]
Was running stable, 0.7.7, updated to 0.7.9 and it seems to work now. I guess running:
`updating git cache: commit date: 2015-08-08` had something to do with it :)

**Comment by radhermit** [comment]
Ah, then https://github.com/pkgcore/pkgcheck/commit/dbed8dd5365ae098f8988d4f5738ea8e2cf52602 or related commits probably fixed things.

**Comment by juippis** [comment]
But then again, now after every `git mv` I seem to get a `MissingMove` error:
```

$ pkgcheck scan --commits
x11-wm/enlightenment
  MissingMove: renamed package: x11-wm/enlightenment -> x11-wm/enlightenment
  IncorrectCopyright: version 0.24.2-r2: incorrect copyright year 2020: '# Copyright 1999-2020 Gentoo Authors'

dev-util/edi
  MissingMove: renamed package: dev-util/edi -> dev-util/edi
  IncorrectCopyright: version 0.8.0-r1: incorrect copyright year 2020: '# Copyright 1999-2020 Gentoo Authors'

dev-libs/efl
  MissingMove: renamed package: dev-libs/efl -> dev-libs/efl
  IncorrectCopyright: version 1.25.1-r3: incorrect copyright year 2020: '# Copyright 1999-2020 Gentoo Authors'
  RequiredUseDefaults: version 1.25.1-r3: profile: 'default/linux/amd64/17.1/desktop/gnome' (23 total) failed REQUIRED_USE: !opengl
  RequiredUseDefaults: version 1.25.1-r3: profile: 'default/linux/amd64/17.1/desktop/gnome' (23 total) failed REQUIRED_USE: gles2-only
  RequiredUseDefaults: version 1.25.1-r10: profile: 'default/linux/amd64/17.1/desktop/gnome' (19 total) failed REQUIRED_USE: !opengl
  RequiredUseDefaults: version 1.25.1-r10: profile: 'default/linux/amd64/17.1/desktop/gnome' (19 total) failed REQUIRED_USE: gles2-only
```
That isn't fixed even by updating the copyright header:
```
dev-libs/efl
  MissingMove: renamed package: dev-libs/efl -> dev-libs/efl
  RequiredUseDefaults: version 1.25.1-r3: profile: 'default/linux/amd64/17.1/desktop/gnome' (23 total) failed REQUIRED_USE: !opengl
  RequiredUseDefaults: version 1.25.1-r3: profile: 'default/linux/amd64/17.1/desktop/gnome' (23 total) failed REQUIRED_USE: gles2-only
  RequiredUseDefaults: version 1.25.1-r10: profile: 'default/linux/amd64/17.1/desktop/gnome' (19 total) failed REQUIRED_USE: !opengl
  RequiredUseDefaults: version 1.25.1-r10: profile: 'default/linux/amd64/17.1/desktop/gnome' (19 total) failed REQUIRED_USE: gles2-only
```

**Comment by radhermit** [comment]
> But then again, now after every `git mv` I seem to get a `MissingMove` error

That's an entirely different bug and was already fixed in 8d88fdd95 (but not in a release yet).

**Comment by juippis** [comment]
Good to know, thanks!
